app/src/main/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentPresenter.kt

```diff
@@ -169,6 +191,48 @@ class PlatformParametersFragmentPresenter @Inject constructor(
     editText.setTag(R.id.platform_parameter_text_watcher, newWatcher)
   }
 
+  private fun handleResetParameter(
+    model: PlatformParameterItemViewModel
+  ) {
+    platformParameterControllerDebugImpl
+      .resetPlatformParameter(model.platformParameterId)
+      .toLiveData()
+      .observe(fragment) { restoredParameterValue ->
+        when (restoredParameterValue) {
+          is AsyncResult.Success -> {
+            resetParameters[model.platformParameterId] = restoredParameterValue.value
+              ?: PlatformParameterValue.getDefaultInstance()
```

Comment:
Can we store the value only when it actually exists, avoiding the use of a default instance that might be 0 or an empty string?
```suggestion
            restoredParameterValue.value?.let { value ->
              resetParameters[model.platformParameterId] = value
            }
```

Reply:
no need now as we have afterResetValue from ephemerals.